index.html

```diff
@@ -24,8 +24,8 @@ <h2 class="quiz-card__question">
           <img src="./bookmark_clear.svg" width="30" height="30" alt="" />
```

Comment:
As discussed with Jerry earlier, use a `button`-element instead of a `div` for the bookmark.

Reply:
Changed as `span`-Element. Also in bookmark.html.